### ISSUE TITLE
extend lexical entry pathways for certain common interactions -- such as: `describe-bindings`

### ISSUE THREAD

**Issue by lansingthomas** [atlas-engineer/nyxt#3011]
Hello dudes,
Easy one today.

**Is your feature request related to a problem? Please describe.**
I find myself entering near-match phrases into the `Execute Command Menu` like "keybinding" or "hotkey", expecting to go to `describe-bindings`, instead I get an empty prompt buffer, or a suggestion for commands with the word "key" like `repeat-key`. 

**Describe the solution you'd like**
I don't fully understand how these fuzzy-search pieces fit together but it might be nice to add task-based colloquial words/phrases to a list somewhere to help people find `describe-bindings` from some other phrases.

Perhaps:
```
key bindings
hotkeys
list hotkeys
keyboard shortcuts
keyboard combos
shortcut keys
```

**Describe alternatives you've considered**
I understand that just looking for "bindings" works, but I would like a solution that prevents others who use different words for this concept from having the issue.

Sensible?
IF not; what am I missing?
Thank you!

**Comment by aartaka** [comment]
Yeah, "bindings" are not perfect. "Keys" are much better, so maybe we should use that more 🤔

**Comment by lansingthomas** [comment]
I suspect that we could go in circles on what is a precise term for the concept `pseudo-macro-keyboard-related-shortened-binding-combo-hotkey-things`

If we point the search results for all terms to the same place it could help. 🤔

**Comment by lansingthomas** [comment]
If i understand correctly, naming the command `describe-keybindings`&rarr;  the fuzzy searching will summon the command for:
```
key bindings
hotkeys
list hotkeys
keyboard shortcuts
keyboard combos
shortcut keys
```
if true, then that seems like a helpful change.

**Comment by lansingthomas** [comment]
Thank you for taking the time to explain. : )

It looks like your first suggestion, rewriting docstring to include the word "key", will shorten navigation for all 6 of the search paths above.

`"Show a buffer with the list of all known bindings for the current buffer."` &rarr; `"Show a buffer with the list of all known key bindings for the current buffer"`

>I can assure you that fuzzy matching won't give you Chipolata, Capocollo or Prosciutto. 😂

this is a shame. If Nyxt could deliver salami we would really revolutionize Internet browsing overnight.

**Comment by lansingthomas** [comment]
if this technique works I foresee myself opening a lot of similar issues. I will not do that, and instead keep a list to batch my updated `docstring` proposals up for efficiency.

**Comment by aartaka** [comment]
> Notice that this can only happen in our next major release (unless we keep both commands in the mean time).

That's an extremely easy one, no?
``` lisp
(setf (fdefinition 'describe-bindings) (fdefinition 'describe-keybindings))
```

**Comment by aartaka** [comment]
If it makes things clearer for someone, then we do. Especially given that we don't really lose anything by adding the name.

**Comment by lansingthomas** [comment]
Thank you both! I love when these simple changes solve problems for people.
